pyxnat/core/derivatives/dtifit.py

```diff
@@ -0,0 +1,14 @@
+XNAT_RESOURCE_NAMES = ['TOPUP_DTIFIT', 'DTIFIT', 'DWI_PREPROCESSING']
+
+
+def download_maps(self, path):
+    '''Downloads a local copy of DWI maps (MD, FA, L1 ~axial, RD)'''
+    import os.path as op
+    rc_files = ['MD', 'FA', 'L1', 'RD']
+    for each in rc_files:
+        res = self._intf.array.experiments(experiment_id=self.parent().id(),
+                                           columns=['subject_label']).data[0]
+        subject_label = res['subject_label']
+        id = res['ID']
+        fp = '%s_%s_%s.nii.gz' % (subject_label, id, each)
```

Comment:
Shadows `id` builtin function, consider simply:
```suggestion
        fp = '%s_%s_%s.nii.gz' % (subject_label, res['ID'], each)
```

Reply:
939f706 fixes this.